typespec-tests/src/main/java/com/encode/bytes/models/Base64urlArrayBytesProperty.java

```diff
@@ -24,7 +24,7 @@ public final class Base64urlArrayBytesProperty implements JsonSerializable<Base6
      * The value property.
      */
     @Generated
-    private List<Base64Url> value;
+    private final List<Base64Url> value;
```

Comment:
Just curious, why this becomes `final`, do we do some fix?

Reply:
This is due to the removal of customization code https://github.com/Azure/autorest.java/blob/abbac504dfb7894d2ed25d13eb46b39b1b98d891/typespec-tests/customization/src/main/java/CustomizationEncodeBytes.java

Customization is no longer needed for this encode/bypes/models test, after switch to azure-json (Alan probably just kept the customization code).

Reply:
I see.